recipe/install.sh

```diff
@@ -0,0 +1,59 @@
+#! /bin/bash
+
+set -e
+
+export EXTRA_CMAKE_ARGS="-GNinja -DCMAKE_INSTALL_LIBDIR=lib -DENABLE_UNSTABLE_API_ABI_HEADERS=ON -DENABLE_LIBCURL=ON -DENABLE_LIBOPENJPEG=openjpeg2"
```

Comment:
Should we enable `zlib` explicitly? See https://gitlab.freedesktop.org/poppler/poppler/-/blob/master/CMakeLists.txt#L80